SU-HSDO/stanford_mrc
# Mrcd8 131 visitor content type

# READY FOR REVIEW

# Summary
- Adding the visitor content type to MRC

# Needed By (Date)
- 12/6/2017

# Urgency
- urgent

# Steps to Test

1. use the mrc_blt branch MRCD8-131-visitor-content-type and update
2. Checkout this branch
3. Enable yearonly module
4. enable MRC visitor
5. Make sure the fields etc are as expected.

# Affected Projects or Products
- MRC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-131
- https://github.com/SU-HSDO/mrc_blt/pull/5
